packages/parser/common_parse_data_full.go

```diff
@@ -625,12 +625,10 @@ func (b *Block) playBlock(dbTransaction *model.DbTransaction) error {
 		)
 		p.DbTransaction = dbTransaction
 
-		if b.GenBlock {
-			err = dbTransaction.Connection().Exec(fmt.Sprintf("SAVEPOINT \"tx-%d\";", curTx)).Error
-			if err != nil {
-				logger.WithFields(log.Fields{"type": consts.DBError, "error": err, "tx_hash": p.TxHash}).Error("using savepoint")
-				return err
-			}
+		err = dbTransaction.Connection().Exec(fmt.Sprintf("SAVEPOINT \"tx-%d\";", curTx)).Error
```

Comment:
why are you use sql queries outside from models package?

Reply:
Moved savepoint functions to db model